react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -194,6 +249,11 @@ class ParticipantView extends Component<Props> {
                     === JitsiParticipantConnectionStatus.ACTIVE)
                 && shouldRenderVideoTrack(videoTrack, waitForVideoStarted);
 
+        // Transform is only enabled for desktop streams rendered in a
+        // large video component (zoomEnabled = true).
+        const transformEnabled
+            = zoomEnabled && getVideoTrackType(videoTrack) === 'desktop';
```

Comment:
This is about my other comment above. We could probably make the decision about the VideoTrack's "fit" property here. WDYT ?